lib/views/create_livestream/active_stream.dart

```diff
@@ -67,31 +81,65 @@ class _ActiveStreamState extends StateAsync<ActiveStream> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: MouseRegion(
-        onHover: setTimer,
-        onExit: mouseOffScreen,
-        child: GestureDetector(
-          onTap: onTap,
-          child: Stack(
-            alignment: Alignment.bottomRight,
-            children: [
-              const _Backdrop(),
-              StreamOverlay(overlayVisible ? 1.0 : 0.25, child: const _ViewCount()),
-              StreamOverlay(overlayVisible ? 1.0 : 0.0, child: const _StreamingCamera()),
-              NavBar.of(context, belowPage: true),
-            ],
-          ),
```

Comment:
When I was testing on an Android device, there was some wonkiness with tap detection, so I switched to a class that will enable a `GestureDetector` or a `MouseRegion`, but not both.